frontend/src/service/tastydate-api-service.ts

```diff
@@ -5,5 +5,14 @@ export interface LoginData {
     password: string | undefined,
 }
 
+export interface RegisterData {
+    name: string | undefined,
+    password: string | undefined,
+    passwordVerify: string | undefined,
```

Comment:
You shure you want to allow undefined as a value?

Reply:
Since you ask: Probably not. I am going to review it and correct it. Thanks!